### PR TITLE
Add dns_vscale.sh

### PR DESCRIPTION
**Pull request by LAV45**
Add support API https://vscale.io

### CODE REVIEW

File: dnsapi/dns_vscale.sh [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  fulldomain=$1
+  txtvalue=$2
+
+  if [ -z "$VSCALE_API_KEY" ]; then
+    VSCALE_API_KEY=""
+    _err "You didn't specify the VSCALE api key yet."
+    _err "Please create you key and try again."
+    return 1
+  fi
+
```
Comment (Neilpang) [inline dnsapi/dns_vscale.sh]: Please add `_saveaccountconf  VSCALE_API_KEY "$VSCALE_API_KEY"`  here,

**Comment by Neilpang** [comment]
Please add our api usage in this readme.md file:
https://github.com/Neilpang/acme.sh/blob/master/dnsapi/README.md

File: dnsapi/README.md [hunk 2]
```diff
@@ -438,6 +438,21 @@ acme.sh --issue --dns dns_infoblox -d example.com -d www.example.com
...
+ 
+First you need to create/obtain API tokens on your [settings panel](https://vscale.io/panel/settings/tokens/).
+ 
+```
+VSCALE_API_KEY="sdfsdfsdfljlbjkljlkjsdfoiwje"
+```
+ 
+Ok, let's issue a cert now:
+```
+acme.sh --issue --dns vscale -d example.com -d www.example.com
```
Comment (Neilpang) [inline dnsapi/README.md]: it should be `--dns  dns_vscale`

Comment (LAV45) [inline dnsapi/README.md]: Ooopss... ))

**Pull request merged**